src/CoreWCF.BuildTools/src/Extensions.cs

```diff
@@ -17,11 +54,11 @@ public static bool IsMatchingParameter(this IParameterSymbol symbol, IParameterS
 
     internal static class SymbolExtensions
     {
-        public static bool HasOneOfAttributes(this ISymbol symbol, params INamedTypeSymbol?[] attributeTypeSymbols)
+        public static (bool Value, AttributeData? AttributeData)  HasOneAttributeOf(this ISymbol symbol, params INamedTypeSymbol?[] attributeTypeSymbols)
         {
```

Comment:
This is a synchronous method so no need to use a named tuple to return multiple values. Just use an out parameter instead. Or you could just return AttributeData as false is only returned when this is null so you could equate returning null as being equivalent to Value being false.

Reply:
done in https://github.com/CoreWCF/CoreWCF/pull/850/commits/dbfa96179c9880b7f43b9fbe8a8e0adf6fab0957